lib/CleantalkSP/SpbctWP/Views/Settings.php

```diff
@@ -118,6 +129,30 @@ private static function getPremiumLink()
         return $link;
     }
 
+    private static function getSupportLink()
+    {
+        global $spbc;
+
+        $support_link = '<a target="_blank" href="https://wordpress.org/support/plugin/security-malware-firewall/">wordpress.org</a>.';
+        if ($spbc->data["wl_mode_enabled"]) {
+            $support_link = '<a target="_blank" href="' . $spbc->data["wl_support_url"] . '">' . $spbc->data["wl_brandname"] . '</a>.';
+        }
+
+        $html = '<span>' . __('Tech support of ' . $spbc->data["wl_brandname"], 'cleantalk') . '&nbsp' . $support_link;
```

Comment:
Better use sprintf here, and please change 'cleantalk' to 'security-malware-firewall'

Reply:
https://github.com/CleanTalk/security-malware-firewall/commit/f00b123939696e04c702b45f9112aa6f10bf0f1e